scripts/build-dist.sh

```diff
@@ -1,11 +1,12 @@
-#!/bin/bash -eo pipefail
+#!/bin/bash 
+
+set -eo pipefail
 
 DBT_PATH="$( cd "$(dirname "$0")/.." ; pwd -P )"
 
 PYTHON_BIN=${PYTHON_BIN:-python}
 
-echo $SCRIPTPATH
```

Comment:
Why was this removed?  I don't see where $SCRIPTPATH is defined, but I assumed it used to output something...

Reply:
I confirmed that this was just echoing a newline in CI currently, it isn't set as env var anywhere that I can tell in the current CI.

https://circleci.com/api/v1.1/project/github/dbt-labs/dbt/46195/output/102/0?file=true&allocation-id=6115a0dab070f52e9c5bdfa7-0-build%2F5A11EAF0

It is the line right before `/home/tox/build_venv//bin/python` aka `$PYTHON_BIN`
```console
  WARNING: The script flake8 is installed in '/home/tox/build_venv/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed backcall-0.2.0 backports.entry-points-selectable-1.1.0 bleach-4.0.0 bump2version-1.0.1 bumpversion-0.6.0 click-8.0.1 decorator-5.0.9 distlib-0.3.2 docutils-0.17.1 execnet-1.9.0 filelock-3.0.12 flake8-3.9.2 flaky-3.7.0 freezegun-0.3.12 iniconfig-1.1.1 ipdb-0.13.9 ipython-7.26.0 ipython-genutils-0.2.0 jedi-0.18.0 matplotlib-inline-0.1.2 mccabe-0.6.1 mypy-0.782 mypy-extensions-0.4.3 parso-0.8.2 pep517-0.11.0 pexpect-4.8.0 pickleshare-0.7.5 pip-tools-6.2.0 pkginfo-1.7.1 platformdirs-2.2.0 pluggy-0.13.1 prompt-toolkit-3.0.19 ptyprocess-0.7.0 py-1.10.0 pycodestyle-2.7.0 pyflakes-2.3.1 pygments-2.9.0 pytest-6.2.4 pytest-dotenv-0.5.2 pytest-forked-1.3.0 pytest-logbook-1.2.0 pytest-xdist-2.3.0 python-dotenv-0.19.0 readme-renderer-29.0 requests-toolbelt-0.9.1 rfc3986-1.5.0 toml-0.10.2 tomli-1.2.1 tox-3.24.1 tqdm-4.62.0 traitlets-5.0.5 twine-3.4.2 typed-ast-1.4.3 virtualenv-20.7.2 wcwidth-0.2.5 webencodings-0.5.1 wheel-0.37.0

/home/tox/build_venv//bin/python
+ rm -rf /home/dbt_test_user/project/dist
+ rm -rf /home/dbt_test_user/project/build
+ mkdir -p /home/dbt_test_user/project/dist
+ for SUBPATH in core plugins/postgres plugins/redshift plugins/bigquery plugins/snowflake
+ rm -rf /home/dbt_test_user/project/core/dist
+ rm -rf /home/dbt_test_user/project/core/build
+ cd /home/dbt_test_user/project/core
+ /home/tox/build_venv//bin/python setup.py sdist bdist_wheel
running sdist
running egg_info
```